src/Orchestration/Container.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Utopia\Orchestration;
+
+class Container {
+    /**
+     * @param string $name
+     * @param string $id
+     * @param string $status
+     * @param array<string, string> $labels
+     */
+    public function __construct(string $name = '', string $id = '', string $status = '', array $labels = [])
```

Comment:
We should add setters and getters here to keep consistency with Utopia libs. All internal attributes should be protected not public.